docs/source/api/federation_resolvers.rst

```diff
@@ -19,6 +19,87 @@
 ``federation_resolvers``
 ************************
 
+``GET``
+=======
+Retrieves :term:`Federation` Resolvers.
+
+:Auth. Required: Yes
+:Roles Required: None
+:Response Type: Array
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| Name       | Required | Description                                                                                         |
+	+============+==========+=====================================================================================================+
+	| id         | no       | Return only the Federation Resolver identified by this integral, unique identifier                  |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| ipAddress  | no       | Return only the Federation Resolver(s) that has/have this IP Address                                |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| type       | no       | Return only the Federation Resolvers of this :term:`Type`                                           |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| orderby    | no       | Choose the ordering of the results - must be the name of one of the fields of the objects in the    |
```

Comment:
I think as we add new documentation we should fix `Choose the ordering of the results - must be the name of one of the fields of the objects in the ``response`` array` as that is not correct

Reply:
We cant do it by lastUpdated

Reply:
My plan was to add a footnote clarifying that the `lastUpdated` field is unavailable for that. Do you think a different wording right in the table would be better?

Reply:
that would be fine

Reply:
I'm working on a branch to do this globally, since the problem exists globally.

Reply:
awesome :) above and beyond 🚀